tests/integration/integration_test.go

```diff
@@ -1004,6 +1005,50 @@ func testConstructOutputValues(t *testing.T, lang string, dependencies ...string
 	}
 }
 
+func TestProviderDownloadURL(t *testing.T) {
+	validate := func(t *testing.T, stdout []byte) {
+		deployment := &apitype.UntypedDeployment{}
+		err := json.Unmarshal(stdout, deployment)
+		assert.NoError(t, err)
+		data := &apitype.DeploymentV3{}
+		err = json.Unmarshal(deployment.Deployment, data)
+		assert.NoError(t, err)
+		urlKey := "pluginDownloadURL"
+		for _, resource := range data.Resources {
+			switch {
+			case providers.IsDefaultProvider(resource.URN):
+				assert.Equal(t, "get.com", resource.Outputs[urlKey])
+			case providers.IsProviderType(resource.Type):
+				assert.Equal(t, "get.pulumi/test/providers", resource.Outputs[urlKey])
```

Comment:
Should we be checking for it in `resource.Inputs` here and above rather than `resource.Outputs`?

Reply:
I think it's probably a good idea to check both.